assets/panel/debugger.properties

```diff
@@ -470,6 +470,10 @@ sourceFooter.unblackbox.accesskey=b
 # with a blackboxed source
 sourceFooter.blackboxed=Blackboxed source
 
+# LOCALIZATION NOTE (sourceFooter.mappedSource): Text associated
+# with a mapped source
+sourceFooter.mappedSource=Source mapped from
```

Comment:
You're implicitly concatenating this with bundleSource, that might not work with all languages.

I see to ways out here:
* Switch this from being a full sentence, and use something like "Source map origin:"
* Use a placeholder in the sentence, so that other languages can move it around as needed (and explain what the placeholder is in the comment): "Source mapped from %S"

Reply:
done. 
in debugger.properties
# LOCALIZATION NOTE (sourceFooter.mappedSource): Text associated
# with a mapped source.
# placeholder %S for source map origin
sourceFooter.mappedSource=(Source mapped from %S)


in Footer.js
{L10N.getFormatStr("sourceFooter.mappedSource", getFilename(bundleSource))}